decidim-generators/spec/runtime/queue_flag_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+if ENV["SIMPLECOV"]
+  require "simplecov"
+
+  SimpleCov.add_filter "/lib/decidim/generators/app_templates/"
+  SimpleCov.add_filter "/lib/decidim/generators/component_templates/"
+end
+
+require "spec_helper"
+require "json"
+require "fileutils"
+require "decidim/gem_manager"
+require "decidim/generators/test/generator_examples"
```

Comment:
I see that these lines are repeates in the different new runtime spec files. Maybe we could try to refactor this to make it DRY? 

I was thinking in something like moving these requirements to `"decidim/generators/test/generator_examples"`, as that's already required for these files and it's only used in these files. What do you think?

Reply:
Fixed in fa9ffe528f